src/ansys/mapdl/core/misc.py

```diff
@@ -82,21 +82,54 @@ def check_valid_routine(routine):
     return True
 
 
+def get_linux_default_ansys_bin(rver):
+    """Find the MAPDL executable using standard Linux installation paths
+
+    Raises:
+        FileNotFoundError: When no binary is found.
+
+    Returns:
+        str: Path to MAPDL executable
+    """
+    for each_path in LINUX_DEFAULT_DIRS:
+        for each_file in [f"ansys{rver}", "mapdl"]:
+
+            ans_root = os.getenv(f"AWP_ROOT{rver}", each_path)
+            mapdlbin = os.path.join(ans_root, f"v{rver}", "ansys", "bin", each_file)
+
+            # rare case where the versioned binary doesn't exist
+            if os.path.isfile(mapdlbin):
+                LOG.debug(f"Found ANSYS binary at {mapdlbin}")
+                return mapdlbin
+            else:
+                LOG.debug(f"NOT found ANSYS binary at {mapdlbin}")
```

Comment:
This seems awkward. Ansys is supposed to be snake case these days except in the actual copyright information.

Reply:
Perfect. Let's change it!